tsconfig.packages.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "composite": true,
     "esModuleInterop": true,
-    "lib": ["ES2020", "DOM"],
+    "lib": ["ES2020", "DOM", "ES2021.WeakRef"],
```

Comment:
Node16 WeakRef

Reply:
If we're targeting node 16 in all of our packages, shouldn't we just be using ES2021?